diff --git a/node_modules/react-native-camera/android/build.gradle b/node_modules/react-native-camera/android/build.gradle
index 4aed495..eeaa075 100644
--- a/node_modules/react-native-camera/android/build.gradle
+++ b/node_modules/react-native-camera/android/build.gradle
@@ -10,7 +10,7 @@ buildscript {
     repositories {
       google()
       mavenCentral()
-      jcenter()
+      mavenCentral()
     }
 
     dependencies {
@@ -56,7 +56,7 @@ android {
 
 repositories {
   google()
-  jcenter()
+  mavenCentral()
   mavenCentral()
   maven { url "https://jitpack.io" }
   maven {
